libqtile/widget/base.py

```diff
@@ -510,7 +510,16 @@ def worker():
             try:
                 text = self.poll()
                 if self.qtile is not None:
+                    # This can throw a runtime error if the event loop is
+                    # not available. This would only happen before qtile is
+                    # started or after it is finalized. The thread here is
+                    # fire and forget, so cannot be signaled.
                     self.qtile.call_soon_threadsafe(self.update, text)
+            except RuntimeError:  # gets raised when the event loop is down
+                logger.debug(
```

Comment:
Can we make this `logger.exception` instead? Then we'll get a stack trace and we can track it down.

Reply:
This only happens when the Thread started on like 527 outlives the event loop, which happens on shutdown. It's debug just to quiet the logs until ThreadedPollText goes away.

Reply:
I mean...the right fix is to make ThreadedPollText go away but that's already done in the dbus PR.

Reply:
I should also note that this is an exception only seen since the lifecycle PR. It's one of the ones we were swallowing before.

Reply:
So if we don't care about these logs, is there any point in logging them? @ramnes?

Reply:
https://github.com/qtile/qtile/pull/2076/files#r553595736